### DISCUSSION TITLE
Search field text obstructed by syntax button

### DISCUSSION THREAD

**Discussion by sambostock** [community/community#9993] [Other Feature Feedback, Questions, & Ideas] [answered]
If the search query is very wide, it becomes obstructed by the question mark syntax info button.

Expected|Actual
-|-
![image](https://user-images.githubusercontent.com/8219340/149017767-a8403297-0474-42f7-9731-f117afc79404.png) the text and button should not occupy the same space|![image](https://user-images.githubusercontent.com/8219340/149017851-e2679511-5311-40a9-b555-cc726cfd77d7.png) the text and button occupy the same space

**Comment by colinwm** [comment, marked as answer]
This is fixed now! Sorry about that.
